app/lib/three_scale/invitation_email_delivery_observer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ThreeScale::InvitationEmailDeliveryObserver
+  def self.delivered_email(message)
+    binding.pry
+    Invitation.find_by(email: message.to)
```

Comment:
btw, use `find` instead of `find_by` here, because if it does not exist, we have a bug that we need to fix.